doc/source/user_guide/visualization.rst

```diff
@@ -1608,6 +1608,13 @@ available in matplotlib. Although this formatting does not provide the same
 level of refinement you would get when plotting via pandas, it can be faster
 when plotting a large number of points.
 
+.. note::
```

Comment:
this is very duplicative of the paragraph above. I would maybe slightly edit that paragraph rather than adding new.

Reply:
The paragraph above is about performance, this note is about correctness. I think it is a good idea to highlight the second, because it can surprise users (it surprised me). So I disagree with you, but then if this is important to you, you could propose an alternative.